runtime.md

```diff
@@ -29,24 +29,59 @@ The root directory to the bundle is provided in the state so that consumers can
 }
 ```
 
-## Lifecycle
+## Typical lifecycle
+
+A typical lifecyle progresses like this:
+
+1. There is no container or running application
+2. A user tells the runtime to start a container+application
```

Comment:
On Wed, Sep 30, 2015 at 11:52:37AM -0700, Mrunal Patel wrote:
> > +2. A user tells the runtime to start a container+application
> 
> start a container for an application.. ?

Without explicit create and start steps, it's “to create a container
and launch an application inside that container”, but that seemed to
long.  Should I just replace “container+application” with “container
and application”?